pkg/utils/paths.go

```diff
@@ -64,13 +64,18 @@ func ExtractPaths(entries map[string]*yang.Entry) ([]*admin.ReadOnlyPath, []*adm
 
 func GetPathValues(prefixPath string, genericJSON []byte) ([]*configapi.PathValue, error) {
 	var f interface{}
+	fmt.Println("---in config models ---GetPathValues ===prefixPath =%s \n genericJSON =%v",prefixPath ,genericJSON)
```

Comment:
`fmt.Println` is not a good way to do logging - there is already a logging library in onos-api that is used for other micro-onos micro-service projects.

I would advise however stepping through this with a debugger on one of the Unit Tests if you are trying to understand what is going on with these functions